src/lib/slack.seed.coffee

```diff
@@ -9,12 +9,14 @@ class Slack
     @url = @composeUrl()
     @timeout = DEFAULT_TIMEOUT
     @maxAttempts = DEFAULT_MAX_ATTEMPTS
+    @webhookOptions = {};
 
   composeUrl: =>
     return "https://slack.com/api/"
 
-  setWebhook: (url) =>
+  setWebhook: (url, options) =>
     @webhookUrl = url
+    @webhookOptions = options if typeof options is "object"
```

Comment:
FWIW - would rather write this as 

```coffeescript
if typeof options is "object"
    @webhookOptions = options
```

It makes it more clear a) there's a conditional, and b) the assignment only occurs if the conditional is true.